invenio_base/app.py

```diff
@@ -168,7 +168,7 @@ def cli(**params):
 
 
 def app_loader(app, entry_points=None, modules=None):
-    """Default application loader.
+    """Return default application loader.
```

Comment:
Right, pydocstyle 2.0.0 wants the imperative verb but I'll change it back to 1.0.0. Thesaurus fun for later